flytekit/models/core/types.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import annotations
+
+import json as _json
```

Comment:
why do we need to import json if it's not used.  also let's move away from the `_` prefix.  i don't think it's ever prevented me from making a bug.

Reply:
sorry, forgot to remove it.